settings.d/dynflow.yml.example

```diff
@@ -0,0 +1,2 @@
+---
+:enabled: false
```

Comment:
Originally, I took the smart_proxy_chef as the scafold for the plugin structure https://github.com/theforeman/smart_proxy_chef/blob/master/settings.d/chef.yml.example, also, all the other features in smart proxy are disabled by default https://github.com/theforeman/smart-proxy/tree/develop/config/settings.d (although the main reason for this was probably because the features are not separated into rpms), I see the majority of other plugins are enabled by default, so I will do the same here